cf_xarray/accessor.py

```diff
@@ -254,10 +254,14 @@ def _get_axis_coord(obj: DataArray | Dataset, key: str) -> list[str]:
         )
 
     search_in = set()
-    if "coordinates" in obj.encoding:
-        search_in.update(obj.encoding["coordinates"].split(" "))
-    if "coordinates" in obj.attrs:
-        search_in.update(obj.attrs["coordinates"].split(" "))
+    coordinates = obj.encoding.get("coordinates", None)
```

Comment:
```suggestion
	attrs_or_encoding = ChainMap(self._obj[name].attrs, self._obj[name].encoding)
    coordinates = attrs_or_encoding.get("coordinates", None)
```

Reply:
I was not sure if it was intentional that here a set of `attrs["coordinates"]` and `encoding["coordinates"]` is being constructed and not the ChainMap operator is being used (which respects then only the first mention of "coordinates" in attrs and encoding).

Reply:
Oh nice catch. I think we should treat coordinates in both attrs and encoding as a bug perhaps.